website/docs/reference/node-selection/syntax.md

```diff
@@ -186,7 +186,7 @@ After issuing one of the above commands, you can reference the results by adding
 
 ```bash
 # You can also set the DBT_STATE environment variable instead of the --state flag.
-dbt run --select "result:<status> --defer --state path/to/prod/artifacts"
+dbt run --select result:<status> --defer --state path/to/prod/artifacts
```

Comment:
Add quotes for the `--select` arguments per [here](https://docs.getdbt.com/reference/node-selection/syntax#specifying-resources) (and https://github.com/dbt-labs/docs.getdbt.com/issues/4040):

```suggestion
dbt run --select "result:<status>" --defer --state path/to/prod/artifacts
```